packages/website/src/pages/form/ActionableItem.tsx

```diff
@@ -1,63 +1,53 @@
-import * as React from 'react';
-import {ActionableItem} from '@coveord/plasma-react';
+import React from 'react';
+import {PageLayout} from '../../building-blocs/PageLayout';
 
-import PlasmaComponent from '../../building-blocs/PlasmaComponent';
+const code = `
+    import * as React from 'react';
+    import {ActionableItem} from '@coveord/plasma-react';
+    
+    export default () => (
+        <ActionableItem
+            id="actionable-item-id"
+            actions={[
+                {value: 'action 1', onOptionClick: () => alert('you triggered the first action')},
+                {value: 'action 2', onOptionClick: () => alert('you triggered the second action')},
+            ]}
+        >
+            click on the dots
+        </ActionableItem>
+    );
+`;
 
-// start-print
-export const ActionableItemExamples = () => (
-    <PlasmaComponent
+const withOnItemClick = `
+    import * as React from 'react';
+    import {ActionableItem} from '@coveord/plasma-react';
+    
+    export default () => (
+        <ActionableItem
+            id="actionable-item-2"
+            onItemClick={(e: React.MouseEvent<HTMLDivElement>) => alert('you triggered the onItemClick method')}
+            actions={[
+                {value: 'action 1', onOptionClick: () => alert('you triggered the first action')},
+                {value: 'action 2', onOptionClick: () => alert('you triggered the second action')},
+            ]}
+        >
+            click on the dots
+        </ActionableItem>
+    );
+`;
+
+export const ActionableItemExamples: React.FunctionComponent = () => (

```

Comment:
You export this component twice (default and named), is that normal?

Reply:
I guess Next doesn't really care but I'll remove that one